crates/control/src/motion/motion_selector.rs

```diff
@@ -51,8 +59,25 @@ impl MotionSelector {
             *context.has_ground_contact,
         );
 
-        self.stand_up_count =
-            stand_up_counting(self.last_motion, current_motion, self.stand_up_count);
+        let is_standing_up =
+            current_motion.is_dispatching() && requested_motion.is_standup_motion();
+
+        let stand_up_count = stand_up_counting(
```

Comment:
I'm not a huge fan of the `ing`, I think this is quite uncommon in our codebase?
```suggestion
        let stand_up_count = stand_up_count(
```
```suggestion
        let stand_up_count = count_stand_ups(
```